src/components/Footer.tsx

```diff
@@ -1,18 +1,38 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from '../store';
+import { StandartItem } from '../types/types';
+import { checkAllItemsCreator, deleteAllItemsCreator } from '../store/actions';
 
-interface Props {
-  amount: number;
+type Props = {
+  checkAll: boolean;
 }
 
-function Footer({ amount }: Props): JSX.Element {
+function Footer({ checkAll }: Props): JSX.Element {
+  const items = useSelector<RootState, StandartItem[]>( state => state.items.items);
+  const amount = items.filter(item => !item.isChecked).length;
+
+  const dispatch = useDispatch();
+  const deleteAllItems = (): void => {
+    dispatch(deleteAllItemsCreator());
+  };
+
 
   return (
     <div className="footer">
       <span className="footer__amount">{amount} items left</span>
       <button className="footer__btn">All</button>
       <button className="footer__btn">Active</button>
       <button className="footer__btn">Completed</button>
-      <button className="footer__btn_last">Clear completed</button>
+      { checkAll
+        ? <button
+          className="footer__btn_last"
+          onClick={deleteAllItems}
+        >Clear completed</button>
+        : <button
+          className="hidden"
+        >Clear completed</button>
+      }
```

Comment:
Here you change only the classname - but you decided to go with ternary operator (too many lines of code). Make it shorter